MiniSim/Menu.swift

```diff
@@ -33,7 +33,40 @@ class Menu: NSMenu {
         self.delegate = self
     }
     
-    func getDevices() {
+    func populateDefaultMenu() {
+        var sections: [DeviceListSection] = []
+        if UserDefaults.standard.enableiOSSimulators {
+            sections.append(.iOS)
+        }
+        
+        if UserDefaults.standard.enableAndroidEmulators {
+            sections.append(.android)
+        }
+    
+        guard !sections.isEmpty else {
```

Comment:
nit: I think simple `if` statement would be better here

Reply:
Agreed :)